package.json

```diff
@@ -605,6 +605,10 @@
 				{
 					"command": "cosmosDB.loadMore",
 					"when": "never"
+				},
+				{
+					"command": "cosmosDB.deleteStoredProcedure",
+					"when": "never"
```

Comment:
Why have the command if we're not shipping stored procedures at this moment?

Reply:
I just did the smallest change I could to hide stored procedures code. I don't wanna completely remove it and change a lot of code